sklearn/linear_model/tests/test_ridge.py

```diff
@@ -815,21 +815,25 @@ def test_n_iter():
 def test_ridge_fit_intercept_sparse():
     X, y = make_regression(n_samples=1000, n_features=2, n_informative=2,
                            bias=10., random_state=42)
+
     X_csr = sp.csr_matrix(X)
 
-    for solver in ['saga', 'sag']:
+    for solver in ['sag', 'sparse_cg']:
```

Comment:
Is there any reason to remove `'saga'`

Reply:
It should raise a warning?

Reply:
yes, saga will rise a warning, because it will be changed to sag solver (see below). This logic didn't change compared to the master. I only make it explicit in this test.